os/src/main/kotlin/com/gojuno/commander/os/Processes.kt

```diff
@@ -120,21 +122,30 @@ private fun prepareOutputFile(parent: File?, keepOnExit: Boolean): File = Random
 
 enum class Os {
     Linux,
-    Mac
+    Mac,
+    Windows
 }
 
-private fun os(): Os {
+internal fun os(): Os {
     val os = System.getProperty("os.name", "unknown").toLowerCase(Locale.ENGLISH)
 
     if (os.contains("mac") || os.contains("darwin")) {
         return Mac
     } else if (os.contains("linux")) {
         return Linux
+    } else if (os.contains("windows")) {
+        return Windows
     } else {
         throw IllegalStateException("Unsupported os $os, only ${Os.values()} are supported.")
     }
 }
 
+internal val Os.nullDeviceFilePath: String
```

Comment:
Maybe return `File` directly? Then it better be a function I guess, to not confuse user of the api about file object sharing